src/biogeochem/DUSTMod.F90

```diff
@@ -756,7 +763,7 @@ subroutine DustEmission (bounds, &
             numer = (u_fld_thr(p)**2.0_r8 - u_impct_thr(p)**2.0_r8 - 2.0_r8 * u_mean_slt(p) * (u_fld_thr(p) - u_impct_thr(p)))
             denom = (2.0_r8 * u_sd_slt(p)**2.0_r8)
             ! Truncate to zero if the expression inside exp is becoming too large
-            if ( numer/denom < 30._r8 )then
+            if ( numer/denom < 30._r8  .and. denom/=0.0_r8)then  ! set numer/denom to be < 30 given exp(30) below is already very large; also set denom to be non-zero (and denom should be autonamtically non-negative given the standard deviation u_sd_slt of the subtimestep wind fluctuation is non-negative)
```

Comment:
Does denom ever actually become zero? If so the order needs to change here so that the check for zero happens first and will short-circuit doing the second test. Otherwise, as it is when DEBUG is on, it'll die with a floating point divide by zero before doing the second check for denom to be zero.

Reply:
Actually I don't think it will ever become zero according the current code setting. u_sd_slt could theoretically be zero, but numerically I have prevented it from happening in line 752 above to avoid infinities in the intermittency scheme.
`u_sd_slt(p) = wnd_frc_slt * (0.001_r8)**0.333_r8   ! should have used 0 theoretically; used 0.001 here to avoid undefined values`
As long as this line 752 exists, `denom/=0.0_r8` is correct. So, yes, `denom/=0.0_r8` is most likely redundant, and it is simply a double-check and does no harm. Should we keep it or remove it?

Reply:
How, about this. We remove it. But, we leave a comment that says why we aren't checking for denom to be zero or negative. If you agree could you do that?

Reply:
Okay! I removed it, edited the comment there, and pushed the change to GitHub.

Reply:
Perfect. Resolving this conversation.